pkgs/development/libraries/libnifalcon/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, cmake, doxygen, libusb1, pkgconfig, expat }:
+
+stdenv.mkDerivation rec {
+  name = "libnifalcon-1.1";
+
+  src = fetchFromGitHub {
+    owner = "libnifalcon";
+    repo = "libnifalcon";
+    rev = "c1546ec8d9a94b3be7d74f8ba9f1a121a0ebb0c2";
+    sha256 = "1gp2cf56rlnlw785vl90w6scpknajw2mdny26yqlx5yl277x9b9s";
+  };
+
+  buildInputs = [ cmake libusb1 doxygen pkgconfig expat ];
```

Comment:
Are these all needed as run-time dependencies?
Build-time dependencies should be declared as `nativeBuildInputs`

Reply:
at least pkg-config needs to move.